src/did/ion-did-resolver.ts

```diff
@@ -0,0 +1,32 @@
+import crossFetch from 'cross-fetch';
+import { DIDMethodResolver, DIDResolutionResult } from './did-resolver';
+
+/**
+ * Resolver for ION DIDs.
+ */
+export class IonDidResolver implements DIDMethodResolver {
+  // cross-platform fetch
+  private fetch = crossFetch;
+
+  /**
+   * @param resolutionEndpoint URL to send DID resolution request to
+   */
+  constructor (private resolutionEndpoint: string) { }
```

Comment:
thoughts on adding the `beta.microsoft` resolution endpoint as a default here?

Reply:
Was thinking to kick the can as more questions would come up like:
1. Are you okay with a straight-up hardcode?
1. Are we going to change to Block's endpoint in the future.
1. `beta.microsoft` itself will not be permanent.

If you are fine with all of the above, then sure, I'll add it.

Reply:
Done: 2cc5b61

Reply:
Done: 2cc5b61